jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh

```diff
@@ -17,7 +17,7 @@ set -e -x
 #
 
 # The versions below don't matter; they are replaced by the Jenkins job
-terra_jupyter_python="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:1.0.9"
```

Comment:
you want to update https://github.com/DataBiosphere/leonardo/blob/develop/jenkins/gce-custom-images/prepare_gce_image.sh instead of this one for your testing